pubgrade/modules/endpoints/builds.py

```diff
@@ -24,10 +27,26 @@
 
 logger = logging.getLogger(__name__)
 
-template_file = '/app/pubgrade/pubgrade/endpoints/kaniko/template.yaml'
+template_file = '/app/pubgrade/modules/endpoints/kaniko/template.yaml'
 BASE_DIR = os.getenv("BASE_DIR")
 if BASE_DIR is None:
     BASE_DIR = '/pubgrade_temp_files'
+gh_action_path="akash2237778/github-actions",
```

Comment:
Again, do not hardcode image names into the code

Reply:
changed to config